tester_app/client/app/bundles/HelloWorld/reducers/index.jsx

```diff
@@ -0,0 +1,10 @@
+import helloWorldReducer from './helloWorldReducer';
+import { $$initialState as $$helloWorldState } from './helloWorldReducer';
```

Comment:
Place comment
// We are changing the name of the import to $$helloWorldState as that will be our export below.

ALTERNATELY and *probably* preferred so it's clear what the initial state is doing.

@alexfedoseev @jbhatab @robwise @patricesf ??

```javascript
import { $$initialState } from './helloWorldReducer';
export default {
  $$helloWorldStore: helloWorldReducer,
};

export const initalStates = {
  $$helloWorldState: $$initialState,
};
```